tuxemon/db.py

```diff
@@ -431,6 +431,9 @@ class TechniqueModel(BaseModel):
     types: Sequence[ElementType] = Field(
         [], description="Type(s) of the technique"
     )
+    usable_in: bool = Field(
```

Comment:
this could probably be clarified some

Reply:
Updated the description: "Whether or not the technique can be used"
Do we need to add an additional # ?

Reply:
> Whether or not the technique can be used _(where?, how?)_

i think that maybe that its not clear that the meaning is supposed to be "Whether or not the technique can be used _outside of combat_".  And "usable_in", is also not clear about this.  Does any of that make sense?

Reply:
Yesterday night it wasn't clear, today it makes sense. I'm going to address it.

Reply:
Ok, I'm going to change **usable_in** in **usable_on**, because it could usable outside the battle **on** something (monsters, items, techniques, world elements, etc.). I hope it makes sense.